source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -276,6 +283,12 @@ def _get_mode_weight(journey):
     return max(mode_weight.get(mode, 1) for mode in journey.tags)
 
 
+def get_min_waiting(journey):
+    """
+    Returns min waiting time in a journey
+    """
+    return min([s.duration for s in journey.sections if s.type == response_pb2.WAITING] or [0])
```

Comment:
Ah no it won't work, the default value need to be +infinity because we want to max this (but then we should rename this function no ?)